internal/uvm/start.go

```diff
@@ -58,11 +55,36 @@ func forwardGcsLogs(l net.Listener) {
 	}
 }
 
+func printToStdout(c net.Conn) {
+	io.Copy(os.Stdout, c)
+}
+
+func processOutput(l net.Listener, doneChan chan struct{}, handler outputHandler) {
+	defer close(doneChan)
+
+	c, err := l.Accept()
+	l.Close()
```

Comment:
Do this before `l.Accept()`

Reply:
Close before Accept?

Reply:
`defer l.Close()`

Reply:
Why though?

Reply:
I re-read the code not on my phone. This is fine